pkg/cronjob/spark_dependencies.go

```diff
@@ -97,7 +97,7 @@ func CreateSparkDependencies(jaeger *v1.Jaeger) *batchv1beta1.CronJob {
 									Resources: commonSpec.Resources,
 								},
 							},
-							RestartPolicy:      corev1.RestartPolicyNever,
+							Volumes:            jaeger.Spec.Volumes,
```

Comment:
I had a comment about this before. The mounting volumes here would not help to get TLS working as the spark dependencies container does not support it. Therefore I would revert this. 

The TLS in spark dependencies is described here https://github.com/jaegertracing/jaeger-operator/issues/294